### ISSUE TITLE
make a new compatible version with mac os x

### ISSUE THREAD

**Issue by jalogisch** [Graylog2/apollo#6]
the current release breaks on a mac os x with the last updates installed

```
﻿﻿1970/01/01 01:00:00.000000 apollo.go:70: �P@Ũ up.
fatal error: unexpected signal during runtime execution
[signal 0xb code=0x1 addr=0x406a4298 pc=0x10bec]

goroutine 5 [running]:
runtime.throw(0x2defe0, 0x2a)
	/usr/local/go/src/runtime/panic.go:527 +0x79 fp=0x1062df44 sp=0x1062df38
runtime.sigpanic()
	/usr/local/go/src/runtime/sigpanic_unix.go:12 +0x4d fp=0x1062df70 sp=0x1062df44
runtime.unlock(0x3e6350)
	/usr/local/go/src/runtime/lock_sema.go:108 +0x10c fp=0x1062df8c sp=0x1062df70
runtime.timerproc()
	/usr/local/go/src/runtime/time.go:208 +0xbb fp=0x1062dfe0 sp=0x1062df8c
runtime.goexit()
	/usr/local/go/src/runtime/asm_386.s:1662 +0x1 fp=0x1062dfe4 sp=0x1062dfe0
created by runtime.addtimerLocked
	/usr/local/go/src/runtime/time.go:116 +0x10c

goroutine 1 [runnable]:
type..hash.net/http.connectMethodKey(0x10643814, 0x2e13ccaf, 0x20)
	/usr/local/go/src/net/http/client.go:1
net/http.(*Transport).getIdleConnCh(0x10672af0, 0x0, 0x106164c0, 0x5, 0x1061ca30, 0x10, 0x0)
	/usr/local/go/src/net/http/transport.go:433 +0x1fa
net/http.(*Transport).getConn(0x10672af0, 0x10672a80, 0x0, 0x106164c0, 0x5, 0x1061ca30, 0x10, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:553 +0x24d
net/http.(*Transport).RoundTrip(0x10672af0, 0x10672a80, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:228 +0x3b3
net/http.send(0x10672a80, 0x307012c8, 0x10672af0, 0x10616540, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:220 +0x404
net/http.(*Client).send(0x1061ac60, 0x10672a80, 0x35, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:143 +0xef
net/http.(*Client).doFollowingRedirects(0x1061ac60, 0x10672a80, 0x30b7e0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:380 +0x9b7
net/http.(*Client).Do(0x1061ac60, 0x10672a80, 0x2bf870, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:175 +0x82
main.readResourceJsonFromNode(0xbffffa7e, 0x20, 0x2bf870, 0x14, 0x0, 0x0, 0x0)
	/Users/lennart/workspace/graylog2/apollo-collector/src/graylog.com/apollo/apollo.go:188 +0xa3
main.readResourceJson(0x2bf870, 0x14, 0x0, 0x0, 0x0)
	/Users/lennart/workspace/graylog2/apollo-collector/src/graylog.com/apollo/apollo.go:215 +0x45
main.main()
	/Users/lennart/workspace/graylog2/apollo-collector/src/graylog.com/apollo/apollo.go:73 +0x301

goroutine 6 [runnable]:
net.Dial(0x28f9f4, 0x3, 0x1061ca30, 0x10, 0x12b5c, 0x106aa004, 0x18, 0x18)
	/usr/local/go/src/net/dial.go:184
net/http.(*Transport).dial(0x10672af0, 0x28f9f4, 0x3, 0x1061ca30, 0x10, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:501 +0x9a
net/http.(*Transport).dialConn(0x10672af0, 0x0, 0x106164c0, 0x5, 0x1061ca30, 0x10, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:596 +0x1613
net/http.(*Transport).getConn.func4(0x10672af0, 0x0, 0x106164c0, 0x5, 0x1061ca30, 0x10, 0x10616580)
	/usr/local/go/src/net/http/transport.go:549 +0x47
created by net/http.(*Transport).getConn
	/usr/local/go/src/net/http/transport.go:551 +0x207
```

**Comment by bernd** [comment]
@jalogisch I released 0.5 using go 1.10. Please test if this works again.

**Comment by bernd** [comment]
Closing this now.
